src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
in this way we use `modifiers`
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--31">
```